modeler.js

```diff
@@ -0,0 +1,131 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+const authorizationConfiguration = {
+  clientId: process.env.MODELER_CLIENT_ID,
+  clientSecret: process.env.MODELER_CLIENT_SECRET,
+  audience: process.env.MODELER_AUDIENCE
+};
+
+// An action that creates a new project.
+async function createProject([projectName]) {
+  // Every request needs an access token.
+  const accessToken = await getAccessToken(authorizationConfiguration);
+
+  // These settings come from your .env file.
+  const modelerApiUrl = process.env.MODELER_API_URL;
+
```

Comment:
```suggestion
  const modelerApiUrl = process.env.MODELER_BASE_URL;
```

This name aligns with what is in the env.example, and what we've done on most other APIs.